### PR TITLE
Gunicorn instrumentation

### PR DESCRIPTION
**Pull request by vaidy4github**

### CODE REVIEW

File: librato_python_web/instrumentor/web/gunicorn_.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+def _worker_notify(f):
+    def decorator(*args, **keywords):
+        response = f(*args, **keywords)
+
+        # Increment the worker count once per minute since that is our rollup interval
+        try:
+            now = int(time.time())
+            threshold = int(60*floor(now/60))
```
Comment (bowillyard) [inline librato_python_web/instrumentor/web/gunicorn_.py]: Is floor() necessary in python, does it convert it to floating point to do the now/60?  Just for my own knowledge.

File: librato_python_web/instrumentor/web/gunicorn_.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            if not __globals.last_reported or __globals.last_reported < threshold:
```
Comment (swilibc) [inline librato_python_web/instrumentor/web/gunicorn_.py]: If you were to initialize `last_reported = 0` then you could simplify this to:
```
            if __globals.last_reported < threshold:
```

**Comment by swilibc** [comment]
Looks good! Merge if you like...

**Comment by swilibc** [comment]
Looks good! Merge if you like...

**Pull request merged**